package/MDAnalysis/coordinates/RDKit.py

```diff
@@ -101,4 +135,113 @@ def __init__(self, filename, **kwargs):
             warnings.warn("No coordinates found in the RDKit molecule")
             coordinates = np.empty((1,n_atoms,3), dtype=np.float32)
             coordinates[:] = np.nan
-        super(RDKitReader, self).__init__(coordinates, order='fac', **kwargs)
\ No newline at end of file
+        super(RDKitReader, self).__init__(coordinates, order='fac', **kwargs)
+
+
+class RDKitConverter(base.ConverterBase):
+    """Convert MDAnalysis AtomGroup or Universe to `RDKit <https://www.rdkit.org/docs/source/rdkit.Chem.rdchem.html#rdkit.Chem.rdchem.Mol>`_ :class:`rdkit.Chem.rdchem.Mol`.
+
+    Example
+    -------
+
+    .. code-block:: python
+
+        import MDAnalysis as mda
+        from MDAnalysis.tests.datafiles import PDB_full
+        u = mda.Universe(PDB_full)
+        mol = u.select_atoms('resname DMS').convert_to('RDKIT')
+    
+
+    .. versionadded:: 2.X.X
+    """
+
+    lib = 'RDKIT'
+    units = {'time': None, 'length': 'Angstrom'}
+
+    def convert(self, obj):
+        """Write selection at current trajectory frame to :class:`~rdkit.Chem.rdchem.Mol`.
+
+        Parameters
+        -----------
+        obj : AtomGroup or Universe
+        """
+        try:
+            from rdkit import Chem
+        except ImportError:
+            raise ImportError('RDKit is required for RDKitConverter but '
+                              'is not installed. Try installing it with \n'
+                              'conda install -c conda-forge rdkit')
+        try:
+            # make sure to use atoms (Issue 46)
+            ag = obj.atoms
+        except AttributeError as e:
+            raise TypeError("No `atoms` attribute in object of type {}, "
+                            "please use a valid AtomGroup or Universe".format(
+                            type(obj))) from e
+
+        mol = Chem.RWMol()
+        atom_mapper = {}
+
+        for atom in ag:
+            try:
+                element = atom.element
+            except NoDataError:
+                # guess atom element
+                # capitalize: transform CL to Cl and so on
+                element = guess_atom_element(atom.name).capitalize()
+            rdatom = Chem.Atom(element)
+            # add properties
+            mi = Chem.AtomPDBResidueInfo()
+            for attr, rdattr in RDATTRIBUTES.items():
+                try: # get value in MDA atom
```

Comment:
Here until 222 is doing quite a lot of things that probably would be worth documenting thoroughly (even if docstring in a private function, at least future developers would know exactly what is going on). How would we feel about moving this code into a private function that get's called within the loop? If anything it might make things more readable (although that could just be me and my tiny laptop screen).